uperf/run.sh

```diff
@@ -14,10 +14,11 @@ scale_out_factor=1   # Determines the number of hosts/nodes that will get used
                  # For ingress/egress: (OCP) total workers = 1 * $scale_out_factor, where uperf client and server
                  #  pods are on the same worker (no external traffic)
 userenv=stream # can be centos7, centos8, stream, rhubi8, debian, opensuse
-osruntime=pod # can be pod or kata for OCP (not yet verified for SRIOV), chroot for remotehost
+osruntime=chroot # can be pod or kata for OCP (not yet verified for SRIOV), chroot for remotehost
```

Comment:
I don't really care, but any particular reason you are changing this default value with these other changes?